server/src/frameworks/shared/crawlDependencies.ts

```diff
@@ -55,13 +55,19 @@ export async function getDependenciesAndPragmas(
     resolvedPath: string;
   }> = [];
   for (const importName of imports) {
-    const resolvedPath = await project.resolveImportPath(
-      sourcePath,
-      importName
-    );
+    try {
+      const resolvedPath = await project.resolveImportPath(
+        sourcePath,
+        importName
+      );
 
-    if (resolvedPath !== undefined) {
-      resolvedImports.push({ importName, resolvedPath });
+      if (resolvedPath !== undefined) {
+        resolvedImports.push({ importName, resolvedPath });
+      }
+    } catch (error) {
+      project.serverState.logger.error(
```

Comment:
Why wouldn't we fail the validation if this fails? Why swallow the error (though reporting it)?

Reply:
Out of band we decided to remove this try/catch.